PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -294,6 +294,21 @@ Although not recommended, if for some reason a user needs to be deleted, follow
 
 To delete the username, click on the OK button. To cancel and keep the username, click on the CANCEL button. This action cannot be undone so proceed with caution.
 
+## Configuring Manager Dashboard
+
+The modules that show up on the manager are cofigurable from the roles settings. To change the dashboard for a for a role, click on the **Settings** tab and click on **Roles**. Scroll down to the **Dashboards** section and here you can add or remove modules.
+
+The new modules for version 1.4 are as follows:
```

Comment:
I don't know if we should call these "modules", as that might be confused with, for instance, the "Primero CP" module. Is there another way we could refer to these? Don't say "v1.4 adds." Just describe the sections of the dashboard and what they show.